src/librustc/middle/stability.rs

```diff
@@ -543,11 +543,6 @@ pub fn check_pat(tcx: &ty::ctxt, pat: &hir::Pat,
         // Foo(a, b, c)
         hir::PatEnum(_, Some(ref pat_fields)) => {
             for (field, struct_field) in pat_fields.iter().zip(&v.fields) {
-                // a .. pattern is fine, but anything positional is
```

Comment:
@Manishearth 
It's an erroneous comment, `PatWildMulti` couldn't be here and `Variant(..)` pattern is `hir::PatEnum(_, None)`